src/main/twirl/gitbucket/core/dashboard/sidebar.scala.html

```diff
@@ -49,10 +49,7 @@
   $('#filter-box').keyup(function(){
     var inputVal = $('#filter-box').val();
     $.each($('li.repo-link a'), function(index, elem) {
-      console.log(inputVal);
-      console.log(elem.text.trim());
-      console.log(elem.text.trim().lastIndexOf(inputVal, 0));
-      if (!inputVal || !elem.text.trim() || elem.text.trim().indexOf(inputVal) >= 0) {
+      if (!inputVal || !elem.text.trim() || elem.text.trim().match(new RegExp(inputVal,'i'))) {
```

Comment:
Use `toLowerCase()` and `indexOf()` instead of `RegExp` to avoid regular expression syntax error.
I fixed your previous pull request in 46896da46e77f5e03a4d907be6f1ada07cb02a36. Please check it.